cli/io/cthon.py

```diff
@@ -0,0 +1,52 @@
+from ceph.ceph import CommandFailed
+from cli import Cli
+
+
+class Cthon(Cli):
+    """This module provides CLI support for cthon tests"""
+
+    def __init__(self, client_node):
+        super(Cthon, self).__init__(client_node)
+        self.client_node = client_node
+
+    def _install_dependencies(self):
+        """Install dependencies for cthon tests"""
+        cmd = (
+            "git clone --depth=1 git://git.linux-nfs.org/projects/steved/cthon04.git;"
```

Comment:
Add library here to clone repo - https://github.com/red-hat-storage/cephci/blob/main/cli/utilities/utils.py#L730